docs/cpp/config-clang-mac.md

```diff
@@ -219,8 +218,7 @@ To debug your code,
 
     ![Hello World Terminal Output](images/clang-mac/helloworld-terminal-output.png)
 
-
-The play button has two modes: **Run C/C++ File** and **Debug C/C++ File**. The default is the last-used mode. If you see the debug icon in the play button, you can just select the play button to debug, instead of selecting the drop-down menu item.
+The play button has two modes: **Run C/C++ File** and **Debug C/C++ File**. The default is the last-used mode. If you see the debug icon in the play button, you can select the play button to debug, instead of selecting the drop-down menu item.
```

Comment:
```suggestion
The play button has two modes: **Run C/C++ File** and **Debug C/C++ File**. The default is the last-used mode. If you see a play button with a bug icon, you can click the play button to debug, instead of selecting the drop-down menu item.
```

Reply:
Prefer "select" over "click" since "select" doesn't imply a mouse (more accessible phrasing)

Reply:
I think I'll leave it as the "debug icon" as we might update the bug in the future